lib/v2/tg/router.js

```diff
@@ -0,0 +1,4 @@
+module.exports = function (router) {
```

Comment:
https://docs.rsshub.app/joinus/advanced/script-standard

- Do not create variations of the same namespace. Use `telegram`, not `tg` or `t`.